permissions/plugin-svn-partial-release-mgr.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "svn-partial-release-mgr"
+paths:
+- "org/jenkins-ci/plugins/svn-partial-release-mgr"
```

Comment:
This path is wrong. Please follow the instructions in this repository's readme, as also linked from the PR template. Both exist for a reason.